src/aat/java/uk/gov/hmcts/cmc/claimstore/tests/functional/LinkDefendantTest.java

```diff
@@ -23,6 +23,7 @@ public void before() {
     }
 
     @Test
+    @Ignore("Disabled due to using CCD")
```

Comment:
This test should be removed when the API is removed, its still in use by the frontend so shouldn't be removed yet